paper/paper.md

```diff
@@ -0,0 +1,354 @@
+tags:
+  - units
+  - quantities
+  - Python
+  - NumPy
+  - sympy
+authors:
+  - name: Nathan J. Goldbaum
+    orcid: 0000-0001-5557-267X
+    affiliation: 1
+  - name: John A. ZuHone
+    orcid: 0000-0003-3175-2347
+    affiliation: 2
+  - name: Matthew J. Turk
+    orcid: 0000-0002-5294-0198
+    affiliation: 1
+  - name: Kacper Kowalik
+    orcid: 0000-0003-1709-3744
+    affiliation: 1
+  - name: Anna Rosen
+    orcid: 0000-0003-4423-0660
+    affiliation: 2
+affiliations:
+  - name: National Center for Supercomputing Applications, University of Illinois at Urbana-Champaign
+    index: 1
+  - name: Harvard-Smithsonian Center for Astrophysics
+    index: 2
+date: 24 May 2018
+bibliography: paper.bib
+---
+
+# Summary
+
+Software that processes data with physical units or that models real world
+quantities with physical units must have some way of managing units. This might
+be as simple as the convention that all floating point numbers are understood to
+be in the same physical unit system (for example, the SI MKS units
+system). While simple approaches like this do work in practice, they also are
+fraught with possible error, both by programmers modifying the code who
+unintentionally misinterpret the units, and by users of the software who must
+take care to supply data in the correct units or who need to infer the units of
+data returned by the software.
+
+The `unyt` library is designed both to aid quick calculations at an interactive
+python prompt and to be tightly integrated into a larger Python application or
+library. To aid quick calculations, the top-level `unyt` namespace ships with a
+large number of predefined units and physical constants to aid setting up quick
+calculations without needing to look up unit data or the value of a physical
+constant. Using the `unyt` library as an interactive calculation aid only
+requires knowledge of basic Python syntax and awareness of a few of the methods
+of the `unyt_array` class - for example, the `unyt_array.to()` method to convert
+data to a different unit. As the complexity of the usage increases, `unyt`
+provides a number of optional features to aid these cases, including custom unit
+registries containing both predefined physical units as well as user-defined
+units, built-in output to disk via the pickle protocol and to HDF5 files using
+the h5py library [@h5py], and round-trip conversions to create units compatible
+with other popular Python unit libraries.
+
+Physical units in the `unyt` class are defined in terms of the dimensions of the
+unit, a string representation, and a floating point scaling to the MKS unit
+system. Rather than implementing algebra for unit expressions, we rely on the
+`SymPy` symbolic algebra library [@SymPy] to handle symbolic algebraic
+manipulation. The `unyt.Unit` object can represent arbitrary units formed out of
+the seven base dimensions in the SI unit system: time, length, mass,
```

Comment:
We took dimensions of "amount" out, remember? So we base everything on 6 of the 7 base units.